### ISSUE TITLE
Custom dataset

### ISSUE THREAD

**Issue by okideal** [hirotomusiker/CLRerNet#41]
Hey. Has anyone tried training with custom dataset?
I manually labeled about 1000 images with lane, disabled crop-preprocess & segmentation loss, modified the max_lanes to 8(num_topk in CollectCLRNet & nms)
The training loss can't converge in 50 epochs, and the image_demo outputs only the 4 weird lanes though my labels have 8.

**Comment by okideal** [comment]
specificly the loss_reg_xytl one, always larger than 50.0

**Comment by MickaMickaMicka** [comment]
we are currently planning to train our own CLRerNet but we are not that far yet. Did you find answers to your questions?
How did you generate the training input data? Did you ensure to generate and use the right format?

In cuLane there is only 4 lanes annotated per frame, and CLRerNet config (and demo code!) might be tuned to that 4 lanes setting? We've checked that CLRerNet has way more than 4 proposals as output, so probably the number 4 comes in the post-processing to ensure that at most 4 lanes are generated.

**Comment by hirotomusiker** [comment]
Exactly, the number of output is important. 
Please refer to the following issue. In the CurveLanes setting, we employ 16-lane output.
https://github.com/hirotomusiker/CLRerNet/issues/19#issuecomment-1916303436
branch:
https://github.com/hirotomusiker/CLRerNet/compare/main...feature/curvelanes

Thank you.
